scipy/signal/tests/test_bsplines.py

```diff
@@ -226,3 +228,33 @@ def test_qspline1d_eval(self):
                       7.32718426, 7.874, 7.81016848, 7.433, 7.03980488, 6.759,
                       6.71900226, 6.203, 4.49418159])
         assert_allclose(bsp.qspline1d_eval(cj, newx, dx=dx, x0=x[0]), newy)
+
+
+def test_sepfir2d_invalid_filter():
+    filt = np.array([1.0, 2.0, 4.0, 2.0, 1.0])
+    image = np.random.rand(7, 9)
+    # No error for odd lengths
+    signal.sepfir2d(image, filt, filt[2:])
+
+    # Row or column filter must be odd
+    with pytest.raises(ValueError, match="odd length"):
```

Comment:
Not: `raises` has already been imported.

Reply:
That was intentional. I'm not a fan of importing everything from the global namespace but also don't want to change code that's not relevant to the test case.